coverage/lcov-report/src/entity/Tenant.ts.html

```
@@ -145,7 +145,7 @@ <h1><a href="../../index.html">All files</a> / <a href="index.html">src/entity</
             <div class='footer quiet pad2 space-top1 center small'>
```

HTMLHint (1.5.0)

[error] The value of attribute [ class ] must be in double quotes.

(attr-value-double-quotes)